src/scripts/services/AuthService.js

```diff
@@ -15,6 +16,7 @@ export default class AuthService {
     });
 
     User.token = response && response.jwt;
+    emitter.emit('authorized');
```

Comment:
The user is only authorized if the response is not null. Move emit under the condition.

Reply:
Done.